[638]  trailing tab in list output (#675)
thin-edge/thin-edge.io: docs/src/references/plugin-api.md

* If an error status is returned, the executable is removed from the list of plugins.
* The list is returned using [CSV with tabulations as separators](https://en.wikipedia.org/wiki/Tab-separated_values).
  Each line has two values separated by a tab: the name of the module then the version of that module.

>   If the version is not present for a module, then the line will contain only the module name without trailing tabulation.

Reviewer: ```suggestion
  If there is no version for a module, then the trailing tabulation is not required and be skipped.
```

Author: fixed
